src/components/OptionMarket/index.tsx

```diff
@@ -135,6 +134,65 @@ export const OptionMarket = () => {
     });
   }
 
+  async function get24HRTotalVolume(assetPoolList: poolByMint) {
+    const keys = Object.keys(assetPoolList);
+    const poolList = keys.map(key => '"' + key + '"');
+    
+    const query = gql`
+      {
+        tokensStats(addresses: [
```

Comment:
This is getting volume for the spot markets. We want the volume for the active PsyOptions markets. 

To get the `ARRAY_OF_SERUM_MARKET_ADDRESSES` you can do the following:

1. Given all OptionMarkets (as loaded from the TVL task), derive their Seurm market addresses
    * See example from PsyOptions UI [here](https://github.com/mithraiclabs/psyoptions-frontend/blob/staging/src/hooks/useDeriveMultipleSerumMarketAddresses.ts#L17-L23)
2. Group the Serum address as follows
_ETH-CALLS_: any `OptionMarket` where the _underlying_asset_mint_ is the ETH mint
_ETH-PUTS_: any `OptionMarket` where the _quote_asset_mint_ is the ETH mint
_BTC-CALLS_: any `OptionMarket` where the _underlying_asset_mint_ is the BTC mint
_BTC-PUTS_: any `OptionMarket` where the _quote_asset_mint_ is the BTC mint
...and so on grouping other unknown mints that show up in a similar fashion, ignoring USDC
3. For each group above query for the Serum stats with the query below (be mindful of rate limits as this list will grow)

```
{
  dailyStats( markets: ARRAY_OF_SERUM_MARKET_ADDRESSES )
    {
    stats {
      au1h
      vol1hUsd
      au24h
      vol24hUsd
      au7d
      vol7dUsd
    }
  }
}
```